jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectorViewPanels.kt

```diff
@@ -559,6 +576,14 @@ class EnvironmentDetailsPanel(private val context: CawsSettings, lifetime: Lifet
         .map { it.toSourceRepository() }
         .sortedBy { it.name }
 
+    private fun isRepo3P(project: CawsProject, repo: String): Boolean {
+        val connectionSettings = context.connectionSettings ?: throw RuntimeException("ConnectionSettings was not set")
+        val url = AwsResourceCache.getInstance().getResource(
+            CawsResources.cloneUrls(CawsCodeRepository(project.space, project.project, repo)), connectionSettings
+        ).toCompletableFuture().get()
+        return url.contains(CawsEndpoints.CAWS_GIT_PATTERN)
```

Comment:
are you trying to make this function return `isRepo3P` or `isRepo1P`?
```suggestion
        return !url.contains(CawsEndpoints.CAWS_GIT_PATTERN)
```